examples/minikube/demo.yaml

```diff
@@ -26,35 +26,27 @@ spec:
         ports:
         - containerPort: 80
 ---
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: v1
+kind: Pod 
 metadata:
   name: app2
+  labels:
+    id: app2
 spec:
-  replicas: 1
-  template:
-    metadata:
-      labels:
-        id: app2
-    spec:
       containers:
```

Comment:
The indentation is off now. I'm surprised that this actually works.

Reply:
It worked before, but I tightened up the indentation anyhow.